### ISSUE TITLE
Detecting game after after failed backup not working

### ISSUE THREAD

**Issue by basxto** [MikeMaximus/gbm#161]
When a backup fails with `Backup aborted.  The saved game path for Fallout Shelter does not exist.`, the next started game does not get detected.
After that it’s working again.

Only tested with proton games so far.

**Comment by MikeMaximus** [comment]
Hmm,  I can't reproduce in Windows.  I don't have access to my Linux machine at the moment to test any Wine/Proton games.

So it successfully detects Game 1, then the backup aborts, and it won't detect the next running game?  At what point does it start working again?  Just after some time has passed?

**Comment by MikeMaximus** [comment]
Alright.  I'll do some more testing as well when I can.

A thought crossed my mind that it might behave strangely if multiple copies of wine64-preloader/wine-preloader are ever running.

**Comment by MikeMaximus** [comment]
I've not had any luck replicating this issue.  Has it happened to you again?

**Comment by basxto** [comment]
No, haven’t observed it again. I haven‘t played proton gameslately.

**Comment by basxto** [comment]
No, haven’t observed it again. I also haven‘t played proton games lately.

**Comment by MikeMaximus** [comment]
Alright, I will close this until something new comes up.  I've been using Proton quite a bit lately to play Subnautica, No Man's Sky and My Summer Car. I haven't encountered this during general use or in any test cases I tried.

**Comment by MikeMaximus** [comment]
I think I discovered what may have been causing this.

While testing the new changes around wine detection in the v118 branch, I noticed it wasn't detecting certain games.  I discovered sometimes the Proton process name is being returned as a full path instead simply "wine64-preloader".  This causes GBM not to see it.

I have no idea why it's doing that today, but it's never done it any time in the past.  In fact, I've never seen any process do that until today.

I'm making a change to assume that the process name may be a full path.
